src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Figure;
+import core.basesyntax.randomizer.FigureSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        final int Array_Length = 6;
```

Comment:
All magic numbers in your code should be constants.
Please see [this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names) to learn about constant fields and their naming requirements.